### PR TITLE
Dev Department Official Statement Regarding Prop 349

### PR DESCRIPTION
**Pull request by rayraspberry**
Draft of Statement Regarding Proposal 349

### CODE REVIEW

**Review by poroburu — DISMISSED** [review]
```suggestion
Charter
```

**Comment by poroburu** [comment]
Sorry, working on my IDE PR code review workflow. It might be a little odd until I figure it out. 
Was not expecting the drop down to push the code suggestion without any preview.

**Comment by kopeboy** [comment]
I see no changes @poroburu, are you about to make them?

**Comment by poroburu** [comment]
> I see no changes @poroburu, are you about to make them?

No changes were made thankfully. 
In VS Code looking for suggest changes, clicking the dropdown ended up requesting a review. Where on GitHub.com they are buttons with a preview 🙀

image

**Review by kopeboy — APPROVED** [review]

File: departments/development/statements/proposal_349/DRAFT-statement-20240510-01.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Development Department Official Statement
+## Re: Juno Commonwealth Proposal #349: Signal proposal for the council to put dapps as the first priority
+#### May 10th, 2024
+
+The Juno Commonwealth Development Department respectfully recommends rejecting Juno Proposal #349, emphasizing the importance of aligning proposed initiatives in a collaborative and good faith with ongoing efforts being carried out by the Development Department.
```
Comment (poroburu) [inline departments/development/statements/proposal_349/DRAFT-statement-20240510-01.md]:
```suggestion
The Juno Charter Development Department respectfully recommends rejecting Juno Proposal #349, emphasizing the importance of aligning proposed initiatives in a collaborative and good faith with ongoing efforts being carried out by the Development Department.
```

Comment (poroburu) [reply departments/development/statements/proposal_349/DRAFT-statement-20240510-01.md]: Suggestion appended onto statement rebase below.

**Comment by poroburu** [comment]
My changes are ready for review now in case the PR still does not like my previous request😺

**Comment by rayraspberry** [comment]
I corrected some grammar and added a line to make clear that communication with Dev Dept should be considered public.

**Review by poroburu — APPROVED** [review]

**Comment by poroburu** [comment]
Re-requesting kopeboy's stale review. LGTM.

**Pull request merged**